### ISSUE TITLE
How to get a relu-activated llama2 model with finetune? any supposed finetune scripts? 

### ISSUE THREAD

**Issue by skykiseki** [SJTU-IPADS/PowerInfer#34]
thx

**Comment by YixinSong-e** [comment]
Thank you very much for your interest in our project. Regarding the specific fine-tuning of relu-llama, the related code and work will still require some time to be ready. If you are very interested, a basic way to reproduce our fine-tuning process is to simply change the activation function from silu to relu and then start training the model on the corpus. You will observe that the loss rapidly decreases around 1 billion tokens, and as the number of training tokens increases, the model's capabilities will be restored more and more. An important thing to note is that our corpus needs to include as many. :)

**Comment by skykiseki** [comment]
> Thank you very much for your interest in our project. Regarding the specific fine-tuning of relu-llama, the related code and work will still require some time to be ready. If you are very interested, a basic way to reproduce our fine-tuning process is to simply change the activation function from silu to relu and then start training the model on the corpus. You will observe that the loss rapidly decreases around 1 billion tokens, and as the number of training tokens increases, the model's capabilities will be restored more and more. An important thing to note is that our corpus needs to include as many. :)

OK.Your achievements are great, and I'm really looking forward to the future ones. 👍
